services/repository-managers/src/main/java/org/sagebionetworks/evaluation/manager/EvaluationManager.java

```diff
@@ -92,5 +93,6 @@ public void deleteEvaluation(UserInfo userInfo, String id)
 
 	EvaluationRoundListResponse getAllEvaluationRounds(UserInfo userInfo, String evaluationId, EvaluationRoundListRequest request);
 
-	void adminMigrateSubmissionQuota(UserInfo userInfo);
+	@WriteTransaction
```

Comment:
We can remove this

Reply:
Also let us add some documentation on what kind of migration we do (e.g. from old submissionQuota to Rounds)